### DISCUSSION TITLE
目录监控（兼容模式）网盘偶尔失效。(希望加上定时执行)

### DISCUSSION THREAD

**Discussion by jinshenganyuci** [jxxghp/MoviePilot#3415] [讨论]
### 确认

- [X] 我的版本是最新版本，我的版本号与 [version](https://github.com/jxxghp/MoviePilot/releases/latest) 相同。
- [X] 我已经 [issue](https://github.com/jxxghp/MoviePilot/issues) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经 [Telegram频道](https://t.me/moviepilot_channel) 中搜索过，确认我的问题没有被提出过。
- [X] 我已经修改标题，将标题中的 描述 替换为我遇到的问题。

### 当前程序版本

1.3.6

### 运行环境

Docker

### 问题类型

主程序运行问题

### 问题描述

我是用的兼容模式。手动运行一次后，可以正常监控识别，但是过了数个小时后就不会自动监控了  
![image](https://github.com/jxxghp/MoviePilot/assets/49135603/69d4810a-f33d-469b-b037-335e1e4f8925)
今天10.04更新的订吞噬星空到网盘指定目录。但是我再MP日志却未发现有监控信息。
![image](https://github.com/jxxghp/MoviePilot/assets/49135603/2c557a68-25d8-42dc-9dab-979a80ef839f)
然后我就手动到MP执行一次目录监控，能识别到
那么从现在开始数小时内 他会自动监控网盘文件。 可能睡一觉他就不监控了
![image](https://github.com/jxxghp/MoviePilot/assets/49135603/012b4582-7456-4535-94f9-c9e59cde83f2)
我的目录监控设置如图
![image](https://github.com/jxxghp/MoviePilot/assets/49135603/e18d0c31-1671-494a-a49a-2145ffe1f86e)
我希望能解决这个问题，所以希望可以增加一个定时功能，我想让他5分钟或者10分钟执行一次。

**Comment by jinshenganyuci** [comment]
今天也是 一样 昨晚凌晨三特点睡觉我测试还能正常监控网盘。  早上10遮天更新  没监控。
![image](https://github.com/jxxghp/MoviePilot/assets/49135603/8c6fe2ea-fc0c-4c42-8293-f1a83993e09e)

日志
![image](https://github.com/jxxghp/MoviePilot/assets/49135603/3d7cf330-2266-47a2-a85b-57d69ffa8af0)

目录状态
![image](https://github.com/jxxghp/MoviePilot/assets/49135603/cba15649-f852-4f94-b77a-039cafceab74)

我现在手动执行后的状态

![image](https://github.com/jxxghp/MoviePilot/assets/49135603/54279f3f-bf78-40d1-8893-73304e5797cb)

**Comment by ershouqingge** [comment]
一样，有时候要手动执行

**Comment by jinshenganyuci** [comment]
> 同样，有时需要手动执行

 只能等了，这次1.3.7还没更

**Comment by YanTaocheng** [comment]
老铁，我用rclone到网盘不能生成mfo等刮削文件，你直接move有么

**Comment by ershouqingge** [comment]
你是什么网盘， 阿里可以通过alist的webdav再rclone，这样mp可以刮削




小严
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "jxxghp/MoviePilot"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年10月29日(星期天) 晚上10:28
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [jxxghp/MoviePilot] 目录监控（兼容模式）网盘偶尔失效。(希望加上定时执行) (Issue #961)



 老铁，我用rclone到网盘不能生成mfo等刮削文件，你直接move有么

**Comment by YanTaocheng** [comment]
我也是阿里云，用alist webdav rclone config到mp的容器，目录监控填的：
/本地下载路劲:/阿里云路径＃rclone_copy

**Comment by jinshenganyuci** [comment]
> 老铁，我用rclone到网盘不能生成mfo等刮削文件，你直接move有么

我没用刮削 我用jellyfin自带的tmdb刮

**Comment by sunbroke555** [comment]
我也是一样。  监控一段时间就失效了。
